mock/config.go

```diff
@@ -0,0 +1,36 @@
+package mock
+
+import (
+	"log"
+	"net"
+	"net/http"
+	"path/filepath"
+
+	"vuvuzela.io/alpenhorn/config"
+)
+
+// ClientAndServer creates a local config server, a config HTTP server, and a
+// config client.
+func ClientAndServer(dir string) (*config.Server, *config.Client) {
```

Comment:
Rename to `LaunchConfigServer` to match other mock package.

Reply:
amended

Reply:
Addressed in https://github.com/vuvuzela/alpenhorn/pull/9 as well